packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4627,7 +4637,14 @@ export default class Meeting extends StatelessWebexPlugin {
         joined = true;
       }
 
-      const mediaResponse = await this.addMedia(mediaOptions, turnServerInfo, forceTurnDiscovery);
+      const mediaResponse = await this.addMediaInternal(
+        () => {
+          return this.joinWithMediaRetryInfo.isRetry ? 'JOIN_MEETING_FINAL' : 'JOIN_MEETING_RETRY';
```

Comment:
nitpick: this can just be ```isRetry ? ... ``` because ```this.joinWithMediaRetryInfo``` is destructured on line  4580

Reply:
hmm not too sure about that one.... the code in the callback is executed later and we need to use the current up-to-date value of `Meeting.joinWithMediaRetryInfo.isRetry` at the point when that callback is called. If we use the `isRetry` closure then I think it will contain the old value, before we've set it to true in line 4700